code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityState.java

```diff
@@ -296,7 +295,7 @@ private boolean isIdentityDirectRegistered(final Map<String, Object> eventHubSha
 		final Map<String, Object> identityDirectInfo = DataReader.optTypedMap(
 			Object.class,
 			extensions,
-			IdentityConstants.SharedState.Hub.EXTENSIONS,
```

Comment:
do we have tests to cover this in the future?

Reply:
Yes there are tests for this case. That was how I found this mistake and used this PR as an opportunity to correct it!
This was a miss during the 2.0 changes.